src/main/java/Stats.java

```diff
@@ -0,0 +1,8 @@
+import java.util.HashMap;
```

Comment:
what is the hashmap imported here for?

Reply:
oops this was because of a previous version, it's not needed anymore because it's not needed anymore